lib/index.js

```diff
@@ -54,9 +57,19 @@ function decorateMethod(method, defaultsOptions) {
 
         let attempts = 0;
 
+        // verbs that exists and doesn't supported in request-promise-native are send in option
+        let rpFunc;
+        if (customVerbs.includes($options.method.toUpperCase())) {
+            $options.method = $options.method.toUpperCase();
```

Comment:
by `request-promise` examples the method is uppercase, but in reality it pass with lowercase as well. whow you prefer i handle this?